frameworks/hdfs/src/main/dist/hdfs_svc.yml

```diff
@@ -0,0 +1,170 @@
+name: {{SERVICE_NAME}}
+principal: {{SERVICE_PRINCIPAL}}
+zookeeper: master.mesos:2181
+api-port: {{PORT0}}
+pods:
+  journal:
+    count: 3
+    tasks:
+      node:
+        goal: RUNNING
+        cmd: "env && ./resolve-dns.sh && ./hadoop-2.6.0-cdh5.7.1/bin/hdfs journalnode"
```

Comment:
Why?  A particular DC/OS release should be tied to a particular HDFS version in my view.  If you want to modify the version, it's an upgrade.